### ISSUE TITLE
Indeterminate ProgressBar ignores Windows animation setting

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#1491]
**Describe the bug**

Indeterminate ProgressBar continues to play its animation even when the user has turned Off "show animations in Windows"

**Steps to reproduce the bug**

1. Go to the Settings app, Ease of Access > Display and toggle Off the "Show animations in Windows" toggle switch
2. Open Xaml Controls Gallery and navigate to the ProgressBar page.
3. Observe that the animation in the first sample continues to play.

**Expected behavior**

Animations shouldn't show when the user has opted out.

**Screenshots**

**Additional context**

There does not appear to exist a specific MAS requirement that's violated with current behavior but respecting user settings seems like the right thing to do.

**Proposed fix**
When animations are turned off, ProgressBar should show a static visual similar to
![image](https://user-images.githubusercontent.com/13142665/68147733-94e22c00-feef-11e9-985f-d52741abfbbb.png)

**Comment by mdtauk** [comment]
How do you demonstrate indeterminate progress without an animation?

**Comment by chingucoding** [comment]
> How do you demonstrate indeterminate progress without an animation?

That is a very good question! What should the ProgressBar behave like without animations? Should it just be blank (and look like a 0% progress)? Or should we render a bar in the middle, which would look a bit like a glitch?

I can't think of a way to show "indeterminate" while having no animation. However showing an animation despite the user requesting to not show one is also not a good solution.

**Comment by shaheedmalik** [comment]
> The Progress bar itself, could still have the value change without any smoothing or transitions. This would not technically be an animated control, but the visual look of it would change over time.
> 
> Would a staggered not smooth "animation" be performant enough to satisfy the user's choice of turning animations off?
> 
> Here is a 4 frame "animation"
> ![image](https://user-images.githubusercontent.com/7389110/67553425-0d353a00-f705-11e9-956b-55e37232ed3c.png)
> 
> No transitions or tweening, just jumping frames.
> 
> And the same could apply to the ProgressRing indeterminate with animations off.

I think the best way is just to display the only the third frame for those who have this setting off.

**Comment by mdtauk** [comment]
> I think the best way is just to display the only the third frame for those who have this setting off.
>
You run the risk of the control looking glitched or broken

**Comment by YuliKl** [comment]
@mdtauk, I hear your concern about not showing any motion, but I think we need to respect the user preference on this one. This is an accessibility setting, likely chosen by users due to specific types of disabilities. I don't think this has anything to do with performance.

I'm inclined to agree with @shaheedmalik and just show a single frame with the colored bar somewhere in the middle-ish of the grey track. I think some loss of visual affordance is acceptable given that the user has opted into this state of Windows. I'm guessing for those users, respecting their preferences is more important than conveying the "something is happening" message.

**Comment by shaheedmalik** [comment]
> > I think the best way is just to display the only the third frame for those who have this setting off.
> 
> You run the risk of the control looking glitched or broken

I mean that what happens when you turn all animations off. 🤷🏾‍♀️

**Comment by mdtauk** [comment]
How does the Win32 control look with animations off?

**Comment by adrientetar** [comment]
On the mac, they have stripe pattern for indeterminate state, which conveys it nicely imo.

![image](https://user-images.githubusercontent.com/1250894/67629008-ca43a580-f86f-11e9-8251-b2db3d7428ac.png)

**Comment by YuliKl** [comment]
> On the mac, they have stripe pattern for indeterminate state, which conveys it nicely imo.
> 
> ![image](https://user-images.githubusercontent.com/1250894/67629008-ca43a580-f86f-11e9-8251-b2db3d7428ac.png)

That is nice. Maybe we can combine @mdtauk's four frames into a static "dashed-line" feel.  The gradient feels a little out of place relative to the other states of the control.

**Comment by adrientetar** [comment]
> The gradient feels a little out of place relative to the other states of the control.

I mean the gradient is reminiscent of Vista-era indeterminate progress bar.

![image](https://user-images.githubusercontent.com/1250894/67724468-5f18e100-f9df-11e9-82b2-60662ed6adeb.png)

So to me at least, it's not unfamiliar.

**Comment by shaheedmalik** [comment]
If not @mdtauk design, maybe something like this?

![image](https://user-images.githubusercontent.com/51347950/67737486-b8820f80-f9e1-11e9-9995-4b48670b2c72.png)

**Comment by mdtauk** [comment]
> If not @mdtauk design, maybe something like this?
> 
> ![image](https://user-images.githubusercontent.com/51347950/67737495-c6d02b80-f9e1-11e9-95d3-6f16d43eadb0.png)

I get the idea behind it, but it does kind of look like a glitch

**Comment by YuliKl** [comment]
> Is this something like what you were thinking?
> 
> ![image](https://user-images.githubusercontent.com/7389110/67742104-f6e3f280-fa12-11e9-9e44-5c174fc8b8d3.png)

Yes. Or maybe
![image](https://user-images.githubusercontent.com/13142665/68147733-94e22c00-feef-11e9-985f-d52741abfbbb.png)

I guess I figured we would keep the indeterminate pill shape the same whether or not it's allowed to animate.
